docs/files/remove-file.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Remove a File
-intro: Removing unused configuration files can ensure that only relevant and necessary information is utilized, thereby reducing the risk of errors during the deployment process.
+intro: Learn how to remove a file when it's no longer needed
```

Comment:
Although theoretically the user can upload any file using Devopness, we do not offer a general file management user experience, so we'd better off focusing on what we do well: manage resource (application, service, ...) configuration files.

```suggestion
intro: Learn how to remove a configuration file when it's no longer needed
```

Reply:
Understood, rewrote most mentions to `files` in the article to `configuration files`.